packages/diff-sequence/diff.js

```diff
@@ -11,277 +16,256 @@ function isObjEmpty(obj) {
   return true;
 }
 
-// ordered: bool.
-// old_results and new_results: collections of documents.
-//    if ordered, they are arrays.
-//    if unordered, they are IdMaps
-DiffSequence.diffQueryChanges = function (ordered, oldResults, newResults,
-                                              observer, options) {
-  if (ordered)
+/**
+ * @function DiffSequence
+ * @description old_results and new_results: collections of documents. if ordered, they are arrays. if unordered, they are IdMaps
+ */
+
+export default DiffSequence = {
```

Comment:
Why `export default`?

Maybe `export const DiffSequence = new class { };` ?

Reply:
You're right. A named export/import will be better for code refactoring in the future... which is the whole point of the PR.

The latest commit also includes a performance optimization to the isObjEmpty method. You can see the performance difference [here](https://jsbench.me/plkon2676h/1).